src/main/webapp/help-workingDir.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  <p>
+    Specify the working directory used for running the command (leave blank for project root)
```

Comment:
If we can't allow an absoulte path, maybe we should specify that the value is relative to the workspace root.